README.md

```diff
@@ -2,145 +2,62 @@
-allowing the developers to use many new language features.
-Currently we do not use TypeScript but plan to look into Flow.
-
-### CSS processor
-[PostCSS](http://postcss.org/) allows to extend the current CSS3 feature set by using several plugins (see package.json).
-
-### Application state
-[Redux](http://redux.js.org/) is used for the global application state. 
-The data which is received through the restful api of the CMS is "cached" and stored in this state container.
-
-### Testing
-* [Jest](https://facebook.github.io/jest/) is used for testing.
-* [<img src="https://d2ogrdw2mh0rsl.cloudfront.net/production/images/static/header/header-logo.svg" width="150">](https://www.browserstack.com) is used for testing cross-browser compatibility
-* To test another api, you can set the new cms domain via Console of your web browser: Just type `window.localStorage.setItem('api-url', 'https://cms-test.integreat-app.de')` and reload the page.
-
-### Linting
-* The linter for JavaScript is [eslint](http://eslint.org/)
-* The linter for CSS is [stylelint](https://stylelint.io/)
-
-You can run the linter by calling **yarn run lint**. Some issues can be fixed automatically by running **yarn run lintfix**
-
-### Type checking
-[Flow](https://flow.org/) is used for static type checking.
-
-Go to Settings > Languages & Frameworks > JavaScript and
-* choose **Flow** as Language version
-* set *Flow package or executable* to **<project_dir>/node_modules/flow-bin**
 
-You can run flow using **yarn flow**. 
-
-## IDE
-**Make sure you have at least [nodejs 6](https://nodejs.org/) installed**
-
-We suggest IntellJ IDEA Ultimate as IDE. Just import this project (from existing sources).
-Run **yarn** in Terminal and right-click on package.json to show the npm scripts. 
-Also mark the *src* and *www* folder as *Source directory*.
-
-Install the following plugins:
-* EJS
-* ESLint
-* React CSS Modules
-* Styled Components
-
-Go to:
+# integreat-webapp
+## Contents
+* [Contributing](#contributing)
+* [Getting Started](#getting-started)
+* [Bug reporting](#bug-reporting)
+* [Pull requests](#pull-requests)
+* [Conventions](docs/01-conventions.md)
+* [Technology Stack](docs/02-technology-stack.md)
+* [Deployment](docs/03-deployment.md)
+
+## Contributing
+If you want to know more about Integreat or if you want to join us, contact [Max](mailto:ammann@integreat-app.de), 
+[Steffen](mailto:kleinle@integreat-app.de) or [our info mail](mailto:info@integreat-app.de) and visit our [website](https://integreat-app.de).
+[Pull requests](#pull-requests) and [bug reports](#bug-reporting) are also very much appreciated.
+ 
+## Getting Started
+### Prerequisites
+* Install [nodejs](https://nodejs.org/). At least v6 is required, but we recommend the v12 LTS.
+Using the latest version (v13) may lead to errors.
+* Install [yarn](https://yarnpkg.com/)
+
+### IDE
+We suggest [IntelliJ IDEA Ultimate](https://www.jetbrains.com/idea/) as IDE. JetBrains provides free licences for students.
```

Comment:
Maybe you could add plugin suggestions here, if you have any :)

Reply:
Plugin suggestions are mentioned below. Apart from those, I personally don't use other plugins (apart from a vim emulation plugin, but I don't consider it a good suggestion here :P)